cluster/agent/cmd/install.go

```diff
@@ -96,14 +109,15 @@ func install(cmd *cobra.Command, args []string) error {
 			log.Println(f)
       
 			err := deployTest(dockerCli, f, "test", 60 /* timeout in seconds */)
-			stack.Remove(dockerCli, stack.RemoveOptions{Namespaces: []string{"test"}})
+			stack.Remove(dockerCli, stack.RemoveOptions{Namespaces: []string{namespace, TEST_NAMESPACE}})
```

Comment:
I don't think you want to remove the amp stack every time you remove the test stack.